### PR TITLE
Help for experimental versions

### PR DESCRIPTION
**Pull request by Fatme**
https://github.com/NativeScript/nativescript-cli/issues/322

### CODE REVIEW

**Comment by dtopuzov** [comment]
+1

**Comment by dtopuzov** [comment]
:+1:

File: resources/help.txt [hunk 1]
```diff
@@ -112,10 +112,21 @@ Usage:
...
     $ tns platform add android [--frameworkPath <File Path>] [--symlink]
-    $ tns platform add ios [--frameworkPath <File Path>] [--symlink] 
+    $ tns platform add android[@<Version>] [--frameworkPath <File Path>] [--symlink]
+    $ tns platform add ios [--frameworkPath <File Path>] [--symlink]
+    $ tns platform add ios[@<Version>] [--frameworkPath <File Path>] [--symlink]
 
 Configures the current project to target the selected platform. 
 
+<Version> is any available version published in npm.
+You can get out of the box 0-day support for native new native platforms.
```
Comment (rosen-vladimirov) [inline resources/help.txt]: "native new native"

**Comment by Fatme** [comment]
@ikoevska

File: resources/help.txt [hunk 2]
```diff
@@ -112,10 +112,21 @@ Usage:
...
 Platform-specific usage:
     $ tns platform add android [--frameworkPath <File Path>] [--symlink]
-    $ tns platform add ios [--frameworkPath <File Path>] [--symlink] 
+    $ tns platform add android[@<Version>] [--frameworkPath <File Path>] [--symlink]
+    $ tns platform add ios [--frameworkPath <File Path>] [--symlink]
+    $ tns platform add ios[@<Version>] [--frameworkPath <File Path>] [--symlink]
 
 Configures the current project to target the selected platform. 
 
+<Version> is any available version published in npm.
```
Comment (ikoevska) [inline resources/help.txt]: \<Version\> is any available version of the respective platform runtime published in npm. If @\<Version\> is not specified, the NativeScript CLI installs the latest stable runtime for the selected platform.
To list all available versions for android, run $ npm view tns-android versions
To list only experimental versions for android, run $ npm view tns-android dist-tags
To list all available versions for ios, run $ npm view tns-ios versions
To list only experimental versions for ios, run $ npm view tns-ios dist-tags

Comment (ikoevska) [inline resources/help.txt]: Let's drop the marketing talk from here. Instead, I'll try incorporating some of it in the readme and the changelog. I'll put up a separate PR for it.

In addition, please, transfer this to the update command as well.

**Comment by ikoevska** [comment]
:+1:

**Pull request merged**